mta-sts.md

```diff
@@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
 
 ~~~~~~~~~
 
+# Appendix 2: Sending MTA State Machine
+
+It is helpful to think of the different states of a sending MTA and a receiving
```

Comment:
Given the pseudocode, I think we should just get rid of this PR. I will delete it if nobody objects.

In particular, after discussion with Laetitia, I'm convinced it's quite a bit simpler than this. As Laetitia pointed out, it is easier to think of it as:

* A sender may (or may not) have a cached (i.e. successfully used before) policy
* A sender may (or may not) have a new policy

A new policy that works (either because it validates or because it is "report-only") can be cached. If it does not "work" we fall back to the cached policy. 

But I'm not sure writing this out is helpful.